lib/start-server.js

```diff
@@ -42,7 +42,10 @@ if (env.versioning.enabled && env.versioning.missingVersionsPage) {
 
 const program = require('commander');
 
-program.option('--port <number>', 'Specify port number').parse(process.argv);
+program
+  .option('--port <number>', 'Specify port number')
+  .option('--live', 'Toggle live reload', {isDefault: true})
```

Comment:
This seems incorrect. The third param of `commander.prototype.options` is either a coercion function or a default value. Giving it an object seems wrong.

http://tj.github.io/commander.js/#Command.prototype.option

Reply:
I believe it should be `.option('--no-live', 'Toggle live reload')` instead. Not sure why, but a default value of true doesn't work when the option is missing in the command. Prefixing with `--no-` will make it `true` when the option is not present.

Could we call it `watch` instead?